### DISCUSSION TITLE
Should the repository be made public?

### DISCUSSION THREAD

**Discussion by cjshearer** [SCCapstone/EZBag#144] [General]
Currently, our repository is private. When it comes time to have another team review and hunt for bugs in our repository, we will need to provide either read-only access to them, or simply make the repository public. I prefer making the repository public as it will allow us to showcase our work to future employers. What are your thoughts @BrendanCReidy @blakete @brendan-curran @cwalmond?

**Comment by BrendanCReidy** [comment]
Public wouldn't be a bad idea.
